app/helpers/application_helper.rb

```diff
@@ -21,4 +21,24 @@ def user_link_target
       edit_email_or_passphrase_user_path(current_user)
     end
   end
+
+  def flash_text_without_email_addresses(message)
+    text_message = strip_tags(message)
+
+    # redact email addresses so they aren't passed to GA
+    text_message.gsub(/[\S]+@[\S]+/, '[email]')
+  end
+
+  SENSITIVE_QUERY_PARAMETERS = %w{reset_password_token invitation_token}
+
+  def has_sensitive_query_parameters?
```

Comment:
I'd prefer if this was named `sensitive_query_parameters?`, prefixing predicates with things like `has_`, `is_`, `can_` or similar is superfluous.